docs/InstallEbpf.md

```diff
@@ -8,20 +8,41 @@ install or update the eBPF installation in the VM.
 
 Do the following from within the VM:
 
-1. Download latest VC++ Redist package from [this location](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170).
-2. Install the redist package downloaded in the above step.
-3. Download the `.msi` file from the [latest release on GitHub](https://github.com/microsoft/ebpf-for-windows/releases).
-4. Execute the `.msi` file you downloaded.
-
-The following components are shown in the MSI to select from:
-
-* **Runtime**: this is the base eBPF runtime, and is required by the other components.  If you select only this
-  component, only [native code generation](NativeCodeGeneration.md) is enabled.
-* **JIT**: this adds support for JIT-compiled eBPF programs and (in a Debug build only) interpreted eBPF programs.
-* **Development**: this adds headers and libraries used for development.  If you only want to use eBPF for development
-  rather than running programs, you can [use the NuGet package](GettingStarted.md#using-ebpf-in-development)
-  instead of the MSI.
-* **Testing**: this adds tests for the eBPF runtime for use by eBPF runtime developers.
+1. Download and install the *VC++ Redist* package from [this location](https://aka.ms/vs/17/release/vc_redist.x64.exe).
```

Comment:
PR description says
> This PR implements a new ebpf-for-windows_installer.msi installer, using the latest [Wix Toolset v3.11.2](https://wixtoolset.org/), and now natively integrates as a standalone and atomic project within the ebpf-for-windows.sln solution, without requiring the 'VC++ Redist' anymore (ref: https://github.com/microsoft/ebpf-for-windows/issues/1478).

So why is this step still needed in the doc?

Reply:
Right, the PR description was not correct (now fixed), the VC++ Redist is required for running the eBPF binaries.